components/constants/webui_url_constants.h

```diff
@@ -41,6 +41,9 @@ extern const char kVPNPanelHost[];
 extern const char kBraveUIWalletPageURL[];
 extern const char kWalletPageHost[];
 #if BUILDFLAG(IS_ANDROID)
+extern const char kWalletPatternPaths[300][100];
+extern const char kWalletPagePaths[300][100];
```

Comment:
This is +60kb size of mostly zeros for app bundle and each process runtime memory usage.